charts/bookkeeper/README.md

```diff
@@ -152,6 +152,7 @@ The following table lists the configurable parameters of the Bookkeeper chart an
 | `image.pullPolicy` | Image pull policy | `IfNotPresent` |
 | `replicas` | Number of bookkeeper replicas | `3` |
 | `maxUnavailableReplicas` | Maximum number of unavailable replicas for bookkeeper PDB | |
+| `upgradeTimeout` | Timeout value for upgrade/rollback | |
```

Comment:
suggestion --> | `upgradeTimeout` | Timeout value for upgrade/rollback in minutes | |

Reply:
done